apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -46,6 +49,11 @@ export function InstructorInstanceAdminSettings({
         title: 'Student Link QR Code',
         content: studentLink,
       })}
+      ${QRCodeModal({
+        id: 'publicLinkModal',
+        title: 'Public Link QR Code',
+        content: publicLink,
+      })}
```

Comment:
Aside: I imagine we've been sort of blindly copying the QR code stuff around, probably from the course instance or assessment settings pages. Is there actual utility here for content sharing? We added this for student-facing assessment/course-instance links thinking that people might put this up on a display during lecture, but I'm not sure when this would be relevant for sharing links.

Reply:
Actually I really do want the QR codes for shared assessments/course instances! It will be great to have for demos and conferences. I never got an email back about some instructional content from someone I met at a conference a few months ago--I really wish they had a QR code in their presentation.

Reply:
Fair enough. I was thinking that while QR codes are neat and all, most applications don't go out of their way to generate them for you. I can buy that they're nice at least for demos.